content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,276 @@ ifndef::include-only-once[]
 include::{root-path}_config.adoc[]
 endif::[]
 
-TODO: Add terms and definitions
+3D Application Terms::
+(To be defined)
 
-Static road network::
-Collection of connected roads enriched by static objects that do not change during runtime of a simulation.
+* Source: 
+* Notes: Taken over from terminology collection list on Sharepoint.
+
+3D Model::
+A (mathematical) geometrical, coordinate-based representation of the surface of an object.
+
+* Source: 
+* Notes: 
+
+3D Environment::
+(To be defined)
+
+* Source: 
+* Notes: 
+
+Asset::
+A 3D geometry combined with assigned material data and specific meta data.
+
+* Source:
+* Notes: 
+
+Camera sensor::
+(To be defined)
+
+* Source: 
+* Notes: 
+
+Closed loop::
+(To be defined)
+
+* Source: 
+* Notes: 
+
+Co-Simulation::
+Simulating multiple independent models connected with interfaces.
+
+* Source: 
+* Notes: 
+
+Environment::
+(To be defined)
+
+* Source: 
+* Notes: Differentiate between 3D environment and simulation environment.
+
+Ground truth::
+True static and dynamic states of all objects as defined by content creator or observed by omniscient observer (e.g. simulation platform).
+
+* Source: 
+* Notes: 
+
+Label::
+(To be defined)
+
+* Source: 
+* Notes: There is no definition for "Label" in OpenLABEL.
+
+Lidar sensor::
+(To be defined)
+
+* Source:
+* Notes:
+
+Material assignment::
+A mapping of material names to parts of 3D models.
```

Comment:
May "... material names or material IDs..."?

Reply:
Would this mapping miss the link to the material properties?

3D model &rarr; material ID/name &rarr; material properties

If so, I would consider this more of a material declaration and providing property values for the name/ID as an assignment, but I might be mislead by the usage of the term "assignment" in computer science.

Reply:
You are right. In my understanding the assignment should contain the link to the actual material properties.